src/Generator/PropertyGenerator.php

```diff
@@ -159,6 +171,47 @@ public function isConst()
         return (bool) ($this->flags & self::FLAG_CONSTANT);
     }
 
+    /**
+     * @param  bool $readonly
+     * @return PropertyGenerator
```

Comment:
Since we are on PHP 7.4+, we can declare these types natively (with `self` and `bool`) - yes, aware that the other methods don't have that :)

Reply:
Added types all over in all files i touched.